src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/GlobalAuthorizationFilterTest.cs

```diff
@@ -67,7 +67,7 @@ public async Task AuthorizationPoliciesCombine()
             var request = new HttpRequestMessage(HttpMethod.Get, "http://localhost/Administration/EitherCookie");
             request.Headers.Add("Cookie", cookie2);
 
-            // Act 2: Will succeed because, with AllowCombiningAuthorizeFilters true, [Authorize] allows either cookie.
+            // Act 2: Will succeed because {Authorize] allows either cookie.
```

Comment:
`[Authorize]`

Reply:
Is there a typo here I can't see?

Reply:
There's a brace `{` at the start instead of `[`, no?

Reply:
Got it